backend/src/main/services/EventService.ts

```diff
@@ -48,15 +76,61 @@ export class EventService {
       throw new HttpException(StatusCodes.BAD_REQUEST, 'Status entered is invalid');
     }
 
-    return this.eventRepository.update(id, eventUpdateDTO);
+    const updatedEvent = await this.eventRepository.update(id, eventUpdateDTO);
+
+    eventUpdateDTO.invitee.forEach(async (invitee) => {
+      await this.emailService.sendEmail(
+        invitee.email,
+        'Meeting Updated',
+        `A meeting you were invited to has been modified: <br><br>
+        Created By: ${eventUpdateDTO.createdBy} <br>
+        Title: ${eventUpdateDTO.title} <br>
+        ${eventUpdateDTO.description ? `Description: ${eventUpdateDTO.description} <br>` : ``}
+        ${eventUpdateDTO.location ? `Location: ${eventUpdateDTO.location} <br>` : ``}
+        Start date: ${new Date(eventUpdateDTO.start).toLocaleString()} <br>
+        End date: ${new Date(eventUpdateDTO.end).toLocaleString()}`
+      );
+    });
+
+    return updatedEvent;
+  };
+
+  public updateInvitedStatus = async (id: number, email: string, accepted: boolean): Promise<number> => {
+    const status: Status = accepted ? Status.ACCEPTED : Status.REJECTED;
+
+    const updatedValue: InvitedDTO = {
+      id: id,
+      email: email,
+      status: status,
+    };
+
+    return this.invitedRepository.update({ id: id, email: email }, updatedValue);
   };
 
   public getEventById = async (id: number): Promise<Event | null> => {
     return this.eventRepository.get(id);
   };
 
   public deleteEvent = async (id: number): Promise<number> => {
-    return this.eventRepository.delete(id);
+    const eventInfo = await this.getEventById(id);
+
+    const deletedEvent = await this.eventRepository.delete(id);
```

Comment:
Should we actually delete the event from the database, or should we add an attribute 'cancelled' to the events table and use that attribute to keep track of cancelled events? @nasaku898 Thoughts?

Reply:
We decided previously that we would be deleting the events.